src/transcripts/episode-21.md

```diff
@@ -0,0 +1,132 @@
se everyone is on their own journey. And so I do this for a living now, but  four years ago probably couldn't have solidly given you an answer for what an ally was. Well, it takes a lot of work and learning. And so for me, I've moved a little bit away from the concept of allyship into like accomplice, because allies can – there's a lot of allies out there and it's complicated, it's complicated to be an ally. And anyone can be an ally to someone else, if you have the power there. So we all have lots of intersecting identities. So like one way that I can be an ally even though I'm an underrepresented woman of color in tech – is I have a lot of political capital being at a company for that long, you accrue capital, you accrue political capital you have through relationships with people who are higher up so when I hear things like I utilize that privilege to go talk to people who are SVPs and GVPs and C-levels about these issues, and so on to me and ally really finds the space that they can slot in and help, and then they don't back out of helping. And so, we an ally skills 101 and 201 training at New Relic and the second one really talks about like, let's be less reactive because you can be an ally in the moment someone says something problematic you can say hey that's not cool or don't do that here or hey that's makes me really uncomfortable. Or you can start doing the things so that's reactive, or you can start doing the things that create really deep cultural structural changes. And that's, that's our ally skills 201 program. And that's really focused on “okay I as an underrepresented person can't decide tomorrow that I don't feel like being underrepresented today. I don't feel like being brown and a tiny woman. I think I'm gonna be a giant white man.” And so, I apply the same concept to allies like if I can't take off my identity and you want to be an ally, I'm going to be an ally all the time, like you can’t step back from that. And when I ask people to move into being accomplices. It's like okay now you're such an ally and you're willing to go down with me in this fight, and not step back when it gets too hard for you. And so, but this is all things and like this is what I tell people about this work, and when I teach ally skills 101 and 201 at New Relic is this stuff changes constantly and you have to stay on the ball, about what this stuff means and you also have to like continue to experience that personal growth. So you can move through the phases of being like, you know, unconsciously incompetent of certain things to, okay I'm consciously incompetent and like working your way through becoming consciously competent, all the time. That's like, active learning, that's actively following leaders on LinkedIn and Twitter, and it's a lot of work, and that's the thing that I don't know that sometimes leadership at companies or people who are like I want to be an ally, like they don't realize that it takes a lot of work, you have to stay current. 
```

Comment:
Not sure if this one will parse correctly, but it seems like there's something weird with **Rachel** here, since GitHub isn't picking up on it.